tests/commands/ironhawk/exists_test.go

```diff
@@ -18,23 +19,32 @@ func TestExists(t *testing.T) {
 			commands: []string{"SET key value", "EXISTS key", "EXISTS key2"},
 			expected: []interface{}{"OK", 1, 0},
 		},
+		// TODO: expected response should be updated for all exists command once multi shard is implemented
 		{
 			name:     "Test EXISTS command with multiple keys",
 			commands: []string{"SET key value", "SET key2 value2", "EXISTS key key2 key3", "EXISTS key key2 key3 key4", "DEL key", "EXISTS key key2 key3 key4"},
-			expected: []interface{}{"OK", "OK", 2, 2, 1, 1},
+			expected: []interface{}{"OK", "OK", 1, 1, 1, 0},
 		},
 		{
 			name:     "Test EXISTS an expired key",
 			commands: []string{"SET key value ex 1", "EXISTS key", "EXISTS key"},
 			expected: []interface{}{"OK", 1, 0},
 			delay:    []time.Duration{0, 0, 2 * time.Second},
 		},
+		// TODO: expected response should be updated for all exists command once multi shard is implemented
 		{
 			name:     "Test EXISTS with multiple keys and expired key",
 			commands: []string{"SET key value ex 2", "SET key2 value2", "SET key3 value3", "EXISTS key key2 key3", "EXISTS key key2 key3"},
-			expected: []interface{}{"OK", "OK", "OK", 3, 2},
+			expected: []interface{}{"OK", "OK", "OK", 1, 0},
 			delay:    []time.Duration{0, 0, 0, 0, 2 * time.Second},
 		},
+		{
+			name:     "EXISTS with no keys or arguments",
+			commands: []string{"EXISTS"},
+			expected: []interface{}{
+				errors.New("wrong number of arguments for 'EXISTS' command"),
+			},
+		},
```

Comment:
Pull from the master and add this test case to `exists_test.go`. EXISTS does not return an error when no arg is passed. It just returns 0. Check the current behaviour and add an apt test.